### DISCUSSION TITLE
Ability to add Cloud or Network drives to library

### DISCUSSION THREAD

**Discussion by hillstead** [deckerst/aves#217] [Ideas]
Be amazing if you could add your Google drive/ dropbox folders to your Library. WebDAV would be great too. Their might be existing flutter plugins to do that I believe.
Great app overall though, really fast and functional

**Comment by deckerst** [comment]
> Be amazing if you could add your Google drive/ dropbox folders to your Library. WebDAV would be great too. Their might be existing flutter plugins to do that I believe.

This has been requested with #126. People there mostly mentioned Nextcloud, but for now it lumps Google Drive, One Drive, etc. all in the same issue. You could show interest in #126 by adding a thumb up reaction to the top post, and subscribe to the issue to get updates from the discussion.

> Great app overall though, really fast and functional

Thanks! Glad you like it.
